app/screens/settings/display/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+import {combineLatest, from as from$, of as of$} from 'rxjs';
+import {switchMap} from 'rxjs/operators';
+
+import {getAllowedThemes, observeConfigBooleanValue} from '@queries/servers/system';
+import {WithDatabaseArgs} from '@typings/database/database';
+
+import DisplaySettings from './display';
+
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
+    const isTimezoneEnabled = observeConfigBooleanValue(database, 'ExperimentalTimezone');
+
+    const allowsThemeSwitching = observeConfigBooleanValue(database, 'EnableThemeSelection');
+    const allowedThemes = from$(getAllowedThemes(database));
```

Comment:
Why do we observe a get, instead of creating an observe function directly?

Reply:
corrected